centreon-awie/packaging/centreon-awie.yaml

```diff
@@ -0,0 +1,27 @@
+name: "centreon-awie"
+arch: "${ARCH}"
+platform: "linux"
+version_schema: "none"
+version: "${VERSION}"
+release: "${RELEASE}${DIST}"
+section: "default"
+priority: "optional"
+maintainer: "Centreon <contact@centreon.com>"
+description: |
+  Move Centreon Awie packaging to nfpm
+  Commit: @COMMIT_HASH@
+vendor: "Centreon"
+homepage: "https://www.centreon.com"
+license: "Apache-2.0"
+
+contents:
+  - src: "../www/modules/centreon-awie/*"
+    dst: "/usr/share/centreon/www/modules/centreon-awie"
+    file_info:
```

Comment:
owner and group must be `apache` for almalinux and `www-data` for debian
currently, nfpm does not support env variable for owner and group, so you should replace it manually like it is done for commit hash : https://github.com/centreon/centreon/blob/develop/.github/actions/package-nfpm/action.yml#L74